docs/pages/config/simple_example.mdx

```diff
@@ -30,13 +49,59 @@ const config = {
       patterns: ['route1', 'route2', 'route3'],
       // For app dir, you need to include route|page, no need to include layout or loading
       // It needs to be prepended with app/ or pages/ depending on the directory used
-      routes: ["app/route1/page", "app/route2/page", "pages/route3"], 
+      routes: ['app/route1/page', 'app/route2/page', 'pages/route3'],
       override: {
-        wrapper: "aws-lambda-streaming",
-      }
+        wrapper: 'aws-lambda-streaming',
+      },
+    },
+  },
+} satisfies OpenNextConfig;
+
+export default config;
+```
+
+## Use aws4fetch instead of AWS sdk
+
+<Callout type="info" emoji="ℹ️">
+  By default we use S3, DynamoDB and SQS for handling ISR/SSG and the fetch
+  cache. We interact with them using AWS sdk v3. This can contribute a lot to
+  the cold start. There is a built-in option to use
+  [aws4fetch](https://github.com/mhart/aws4fetch) instead of the AWS sdk that
+  can reduce cold start up to 300ms. Requires `OpenNext v3.0.3+`. Here is how
+  you enable it:
+</Callout>
+
+```ts
+import type { OpenNextConfig } from 'open-next/types/open-next';
+const config = {
+  default: {
+    override: {
+      tagCache: 'dynamodb-lite',
+      incrementalCache: 's3-lite',
+      queue: 'sqs-lite',
+    },
+  },
+} satisfies OpenNextConfig;
+
+export default config;
+```
+
+## Running in Lambda@Edge
+
+<Callout type="info" emoji="ℹ️">
+  External middleware is not supported in Lambda@Edge.
```

Comment:
That's not correct actually, in sst you cannot use external middleware with another function deployed in lambda@edge, because it would mean 2 function deployed at the same place.
External middleware is deployed in lambda@edge by default.
We can just remove this, it's more a sst stuff than an OpenNext one